rs/nns/governance/src/governance/disburse_maturity.rs

```diff
@@ -40,7 +40,8 @@ const DISBURSEMENT_DELAY_SECONDS: u64 = ONE_DAY_SECONDS * 7;
 /// The maximum number of disbursements in a neuron. This makes it possible to do daily
 /// disbursements after every reward event (as 10 > 7).
 const MAX_NUM_DISBURSEMENTS: usize = 10;
-/// The minimum amount of ICP to disburse in a single transaction.
+/// The minimum amount of ICP that need to be minted when disbursing maturity. A neuron can only disburse an amount of maturity that results in minting at least this many ICP (in e8) assuming the worst case maturity modulation.
```

Comment:
Can you explain why a limit is needed?

Maybe, also explain why/how this value was chosen. E.g. what harm would be allowed if users were allowed to disburse 0.1 ICP?

And why involve maturity modulation? Seems like extra complexity.